server/lobbyconnection.py

```diff
@@ -964,6 +964,12 @@ async def command_game_join(self, message):
         try:
             game = self.game_service[uuid]
         except KeyError:
+            await self.send({
+                "command": "game_join_failed",
+                "reason": "HOST_LEFT_GAME",
```

Comment:
The only other thing I'm thinking now is if maybe these codes should be lowercase. I think everything else in the protocol is generally lowercase.